.github/actions/README.md

```diff
@@ -35,6 +35,10 @@ Quality assessment and quality control. Standardizes formatting including spell
 #### [repo2docker.yaml](../workflows/repo2docker.yaml)
 [Build a Docker image](https://github.com/jupyterhub/repo2docker-action) for JupyterHub/BinderHub
 
+#### [template-sync.yaml](../workflows/template-sync.yaml)
+Open a PR to update the templated repo to incorporate changes made to the
+[template repo](https://github.com/uwhackweek/jupyterbook-template).
+
 #### [test.yaml](../workflows/test.yaml)
 Build the websites (JupyterBook and front page). Run on Pull Requests against every commit and via a 'cron' schedule to maintain caching [since otherwise the cache expires if untouched in 7 days](https://docs.github.com/en/actions/advanced-guides/caching-dependencies-to-speed-up-workflows#usage-limits-and-eviction-policy)
```

Comment:
Anyone know where this action went? repo2docker-PR.yaml is also not documented here.

Reply:
I renamed "test" to build_website.yaml and (naughtily) did not update the docs
8e96762a61099b2a113ba72e681c4cc6ce295777